test/scripts/HarvesterUSDATest.t.sol

```diff
@@ -0,0 +1,226 @@
+import { CollateralSetupProd } from "contracts/transmuter/configs/ProductionTypes.sol";
+import { ITransmuter } from "interfaces/ITransmuter.sol";
+import "utils/src/Constants.sol";
+import { IERC20 } from "oz/interfaces/IERC20.sol";
+import { IAgToken } from "interfaces/IAgToken.sol";
+import { Harvester } from "contracts/helpers/Harvester.sol";
+
+import { RebalancerFlashloan, IERC4626, IERC3156FlashLender } from "contracts/helpers/RebalancerFlashloan.sol";
+
+interface IFlashAngle {
+    function addStablecoinSupport(address _treasury) external;
+
+    function setFlashLoanParameters(address stablecoin, uint64 _flashLoanFee, uint256 _maxBorrowable) external;
+}
+
+contract HarvesterUSDATest is Test {
+    using stdJson for string;
+
+    ITransmuter transmuter;
+    IERC20 USDA;
+    IAgToken treasuryUSDA;
+    IFlashAngle FLASHLOAN;
+    address governor;
+    RebalancerFlashloan public rebalancer;
+    uint256 ethereumFork;
+    Harvester harvester;
+    uint64 public targetExposure;
+    uint64 public maxExposureYieldAsset;
+    uint64 public minExposureYieldAsset;
+
+    function setUp() public {
+        ethereumFork = vm.createSelectFork(vm.envString("ETH_NODE_URI_MAINNET"), 19939091);
+
+        transmuter = ITransmuter(0x222222fD79264BBE280b4986F6FEfBC3524d0137);
+        USDA = IERC20(0x0000206329b97DB379d5E1Bf586BbDB969C63274);
+        FLASHLOAN = IFlashAngle(0x4A2FF9bC686A0A23DA13B6194C69939189506F7F);
+        treasuryUSDA = IAgToken(0xf8588520E760BB0b3bDD62Ecb25186A28b0830ee);
+        governor = 0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8;
+        rebalancer = RebalancerFlashloan(0x22604C0E5633A9810E01c9cb469B23Eee17AC411);
+        targetExposure = uint64((15 * 1e9) / 100);
+        maxExposureYieldAsset = uint64((80 * 1e9) / 100);
+        minExposureYieldAsset = uint64((5 * 1e9) / 100);
+
+        harvester = new Harvester(
+            address(rebalancer),
+            address(STEAK_USDC),
+            targetExposure,
+            1,
+            maxExposureYieldAsset,
+            minExposureYieldAsset,
+            1e8
+        );
+
+        vm.startPrank(governor);
+        transmuter.toggleWhitelist(Storage.WhitelistType.BACKED, NEW_DEPLOYER);
+        transmuter.toggleTrusted(governor, Storage.TrustedType.Seller);
+        transmuter.toggleTrusted(address(harvester), Storage.TrustedType.Seller);
+
+        vm.stopPrank();
+    }
+
+    function testUnit_Harvest_IncreaseUSDCExposure() external {
+        // At current block: USDC exposure = 7.63%, steakUSDC = 75.26%, bIB01 = 17.11%
+        (uint256 fromUSDC, uint256 total) = transmuter.getIssuedByCollateral(address(USDC));
+        (uint256 fromSTEAK, ) = transmuter.getIssuedByCollateral(address(STEAK_USDC));
+        harvester.harvest(USDC);
+        (uint256 fromUSDC2, uint256 total2) = transmuter.getIssuedByCollateral(address(USDC));
+        (uint256 fromSTEAK2, ) = transmuter.getIssuedByCollateral(address(STEAK_USDC));
+        assertGt(fromUSDC2, fromUSDC);
```

Comment:
can we add an estimation to where we will be 
Even with fees and wrong oracle we should be at most 1% (should be less if oracle deviation is 0.3% and fees less than 0.3%) to target